content/en/docs/ct-logs.html

```diff
@@ -11,7 +11,15 @@
 
 <p><a href="https://www.certificate-transparency.org/what-is-ct">Certificate Transparency (CT)</a> is a system for logging and monitoring the issuance of TLS certificates. CT greatly enhances everyone's ability to monitor and study certificate issuance, and these capabilities have led to numerous improvements to the CA ecosystem and Web security. As a result, CT is rapidly becoming critical infrastructure.</p>
 
-<p>Let's Encrypt submits all certificates we issue to CT logs. We also operate a CT log ourselves, which all publicly trusted certificate authorities are welcome to submit to.</p>
+<p>Let's Encrypt submits all certificates we issue to CT logs. We also operate several CT logs ourselves, which all publicly trusted certificate authorities are welcome to submit to.
+
+<ul>
+    <li><a href="https://letsencrypt.org/2019/05/15/introducing-oak-ct-log.html">Oak</a> is our production log and will eventually be incorporated in browser CT programs.<p></li>
```

Comment:
There is a `<p>` alone here; that's why the build fail: https://travis-ci.org/letsencrypt/website/builds/646092003#L273

To run local tests you can: `npm install && npm run build && npm test` (it will change with #864)

Reply:
Ah, I ran `npm install && npm run build; hugo server -F`. Thanks!

Reply:
`npm test` runs a few more tests, for example, it check if the html **generated** by hugo is valid 🙂 (and a few other things on css and js) - it must be run after guho build the website (not in server mode). But anyway travis is here to catch it 😉